jdaviz/configs/imviz/wcs_utils.py

```diff
@@ -389,52 +377,22 @@ def _prepare_rotated_nddata(image_data, wcs, rotation_angle, refdata_shape):
 
     # create a GWCS centered on ``filename``,
     # and rotated by ``rotation_angle``:
-    new_rotated_gwcs = rotated_gwcs(
+    new_rotated_gwcs = _rotated_gwcs(
         central_world_coord, rotation_angle, pixel_scales, cdelt_signs
     )
 
-    # create an all-nan NDDataArray with the rotated GWCS:
-    ndd = NDDataArray(
-        data=np.nan * np.ones(refdata_shape),
+    # create a fake NDData with the rotated GWCS:
+    ndd = NDData(
+        data=np.empty(refdata_shape, dtype=np.int8),

```

Comment:
I used NaNs here to be sure that if a user finds a way to extract those data, they aren't ever misinterpreted as something meaningful. Is there a motivation to use `empty` instead?

Reply:
I'd advocate for revert. I agree that if you see this array in a notebook, it's clear something funny is going on. My concern is that if one of these layers were naively plotted inside jdaviz or outside in mpl for example, it would look like a uniform near-zero image. Since all values are about the same, you wouldn't be able to see the separate pixels and notice that there are only 2x2 pixels. So the nans are just meant to signal "this isn't the data that you want" by one more means.

Reply:
Actually, you cannot see "nan" in Imviz either (so it is basically setting values to zeros as far as visibility is concerned). It is only obvious when you mouse over one of the 4 pixels that the value is "nan". In fact, any homogenous array won't show (even when I use `np.ones`). So if the goal is for user to see immediately they are looking at a 2 x 2 array that cannot be real, I would suggest `np.arange(4).reshape((2, 2))`.